submissions/Zakkarat/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,78 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+  name: "Paolo",
+  species: "dog",
+  gender: 'male',
+  legs: 4,
+  hands: 0,
+  saying: "Bark!"
+}
+const cat = {
+  name: "Sophie",
+  species: "cat",
+  gender: 'female',
+  legs: 4,
+  hands: 0,
+  saying: "Mur-meow?"
+}
+const human = {
```

Comment:
In accordance with the technical specification you must define dog, cat, woman and man. Also cat-woman is optional. I think you must define `man` here.

Reply:
Done!